packages/xo-server-audit/src/index.js

```diff
@@ -30,8 +30,11 @@ const DEFAULT_BLOCKED_LIST = {
   'cloudConfig.getAll': true,
   'cloudConfig.getAllNetworkConfigs': true,
   'group.getAll': true,
+  'host.getBiosInfo': true,
   'host.getIpmiSensors': true,
+  `host.getMdadmHealth': true,
```

Comment:
Why do we want to hide the error of this method?
`XENAPI_MISSING_PLUGIN` and `UNKNOWN_XENAPI_PLUGIN_FUNCTION` are caught so they don't throw an error. So if an error is thrown, that's probably something we want to know. Right?